src/components/dashboard/overview-section/notification/notification.tsx

```diff
@@ -0,0 +1,75 @@
+import { NotificationExpense } from "./notification-expense";
+import { NotificationInvitation } from "./notification-invitation";
+import { NotificationType } from "../../../../@types/notification";
+import "./notification.css";
+
+const notifications: NotificationType[] = [
+  {
+    type: "expense",
+    props: {
+      avatarSrc: "",
+      expenseDescription: "Coffee",
+      listName: "Office Expenses",
+      amount: 100,
+    },
+  },
+  {
+    type: "expense",
+    props: {
+      avatarSrc: "",
+      expenseDescription: "Groceries",
+      listName: "Office Expenses",
+      amount: 100,
+    },
+  },
+  {
+    type: "invitation",
+    props: {
+      avatarSrc: "",
+      listName: "Project X",
+      responses: [{ accepted: true }, { accepted: false }],
+    },
+  },
+  {
+    type: "expense",
+    props: {
+      avatarSrc: "",
+      expenseDescription: "Utilities",
+      listName: "Office Expenses",
+      amount: 100,
+    },
+  },
+];
+
+export const Notification = () => (
+  <div className="notifications-container">
+    {notifications.map((notification, index) => {
```

Comment:
[Why Index as a key is an anti-pattern](https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067)